.packit.yaml

```diff
@@ -0,0 +1,214 @@
+    upstream_tag_template: python-copr-{version}
+
+  python-copr-common:
+    downstream_package_name: python-copr-common
+    upstream_package_name: copr-common
+    paths:
+      - ./common
+    specfile_path: python-copr-common.spec
+    files_to_sync:
+      - python-copr-common.spec
+    upstream_tag_template: python-copr-common-{version}
+
+  copr-backend:
+    downstream_package_name: copr-backend
+    upstream_package_name: copr-backend
+    paths:
+      - ./backend
+    specfile_path: copr-backend.spec
+    files_to_sync:
+      - copr-backend.spec
+    upstream_tag_template: copr-backend-{version}
+
+  copr-dist-git:
+    downstream_package_name: copr-dist-git
+    upstream_package_name: copr-dist-git
+    paths:
+      - ./dist-git
+    specfile_path: copr-dist-git.spec
+    files_to_sync:
+      - copr-dist-git.spec
+    upstream_tag_template: copr-dist-git-{version}
+
+  copr-keygen:
+    downstream_package_name: copr-keygen
+    upstream_package_name: copr-keygen
+    paths:
+      - ./keygen
+    specfile_path: copr-keygen.spec
+    files_to_sync:
+      - copr-keygen.spec
+    upstream_tag_template: copr-keygen-{version}
+
+  copr-messaging:
+    downstream_package_name: copr-messaging
+    upstream_package_name: copr-messaging
+    paths:
+      - ./messaging
+    specfile_path: copr-messaging.spec
+    files_to_sync:
+      - copr-messaging.spec
+    upstream_tag_template: copr-messaging-{version}
+
+  copr-rpmbuild:
+    downstream_package_name: copr-rpmbuild
+    upstream_package_name: copr-rpmbuild
+    paths:
+      - ./rpmbuild
+    specfile_path: copr-rpmbuild.spec
+    files_to_sync:
+      - copr-rpmbuild.spec
+    upstream_tag_template: copr-rpmbuild-{version}
+    actions:
+      post-upstream-clone:
+        - sh -c "wait-for-copr --owner $COPR_OWNER --project $COPR_PROJECT python-copr `git rev-parse --short HEAD`"
+
+  copr-selinux:
+    downstream_package_name: copr-selinux
+    upstream_package_name: copr-selinux
+    paths:
+      - ./selinux
+    specfile_path: copr-selinux.spec
+    files_to_sync:
+      - copr-selinux.spec
+    upstream_tag_template: copr-selinux-{version}
+
+  copr-cli:
+    downstream_package_name: copr-cli
+    upstream_package_name: copr-cli
+    paths:
+      - ./cli
+    specfile_path: copr-cli.spec
+    files_to_sync:
+      - copr-cli.spec
+    upstream_tag_template: copr-cli-{version}
+    actions:
+      post-upstream-clone:
+        - sh -c "wait-for-copr --owner $COPR_OWNER --project $COPR_PROJECT python-copr `git rev-parse --short HEAD`"
+
+  copr-frontend:
+    downstream_package_name: copr-frontend
+    upstream_package_name: copr-frontend
+    paths:
+      - ./frontend
+    specfile_path: copr-frontend.spec
+    files_to_sync:
+      - copr-frontend.spec
+    upstream_tag_template: copr-frontend-{version}
+    actions:
+      post-upstream-clone:
+        - sh -c "wait-for-copr --owner $COPR_OWNER --project $COPR_PROJECT python-copr `git rev-parse --short HEAD`"
```

Comment:
What if `python-copr` isn't actually changed by the PR?

Reply:
currently packit builds everything even no change is done to the package (there's issue about it https://github.com/packit/packit-service/issues/2006)

for that reason we have now only enabled builds for merging to main branch. Triggering builds in PRs are done manually via `/packit build` command